### PR TITLE
Change dev server recommendation in README

### PR DESCRIPTION
**Pull request by skddc**
... and link to servers page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,12 +22,13 @@ servers and syncing data across devices and applications.
 
 To develop remoteStorage-enabled apps, you need to have a
 remoteStorage-compatible storage account. We recommend
-[reStore](https://github.com/jcoglan/restore) for running a local test server.
```
Comment (untitaker) [inline README.md]: Can't we just remove all recommendations from the README, and move them into the wiki?

Comment (skddc) [inline README.md]: What page though? The "servers" page isn't just focused on developers, so I think the README that's contained in the repo is a good place to give that pointer. The next paragraph there points to the wiki.

Comment (untitaker) [inline README.md]: Good point about dev-specific recommendation.

**Comment by untitaker** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (skddc) [inline README.md]: Also good point about the wiki, though. :) I think we should replicate that at least in the beginner's guide, if it isn't there yet.

**Comment by galfert** [comment]
+1

**Pull request merged**